surface/surface.proto

```diff
@@ -88,4 +88,6 @@ message Model {
     string name = 1; // a free-form title for the API
     repeated Type types = 2; // the types used by the API
     repeated Method methods = 3; // the methods (functions) of the API
+    repeated string symbolicReferences = 4; // references to other OpenAPI files
+    string sourceName = 5; // source name of the OpenAPI description (.yaml or .json)
```

Comment:
please use snake_case for such compound names (will probably need to regenerate the Go file, but it shouldn't break your code).

Sorry I missed this earlier.

Reply:
Done. Other fields inside surface.proto are still cameCase. Should I change them as well?

Reply:
Eh no, that's ok. I just prefer not to continue to add to it. But that's just me. Even if we did, it would be in a different PR.